system/core/base.php

```diff
@@ -482,6 +482,15 @@ function hypha_addPage($type, $language, $name, $private) {
 		return hypha_setPage($newPage, $language, $name, $private);
 	}
 
+	function hypha_deletePage($id) {
```

Comment:
Should this have a comment saying it only modifies hypha.xml and you should usually use deletePage instead?

Reply:
From a scope perspective I think it's a bit weird that `hypha_deletePage` "knows" about `deletePage`. I think the readme should contain a chapter containing datatypes and this information should be placed there.